.azure-pipelines/BranchProtectionForPrivateRepo.yml

```diff
@@ -1,6 +0,0 @@
-jobs:
```

Comment:
I think this one is still used in the private repo. Can you maybe replace it with an action or something to help tell folks not to merge into private main?

Reply:
The check is run in private, but it's not required.  And I think the "Merging is blocked" message from GH branch protection is sufficient.